src/js/components/search/visualizations/rank/chart/ChartGroup.jsx

```diff
@@ -42,14 +44,26 @@ export default class ChartGroup extends React.Component {
         }
     }
 
+    processLink(label) {
+        let title = label;
+
+        if (this.props.linkID !== '') {
+            title = (<a href={`${this.props.urlRoot}${this.props.linkID}`}>
```

Comment:
Can you do this directly inside a `<text>` SVG element? It works on Chrome but I'm not sure that's standard behavior.